distributed/deploy/cluster.py

```diff
@@ -0,0 +1,64 @@
+class Cluster(object):
+    scheduler : Scheduler
+
+    @property
+    def scheduler_address(self):
+        """ This what Client(...) currently looks for to find an address """
+        return self.scheduler.address
+
+    def scale_up(self, n: int) -> List[T]:
+        """ Scale up the cluster to at least n workers
+
+        This is mostly intended to be used with Adaptive.  See scale for a more
+        general user-facing function.
+
+        Parameters
+        ----------
+        n: int
+            Total number of workers to have after calling this function
+
+        Returns
+        -------
+        A list of the newly created workers, represented in whatever way makes
+        sense for the cluster
+
+        See Also
+        --------
+        scale: more user facing
+        """
+        pass
+
+    def scale_down(self, workers: Union[List[T], List[str]]):
```

Comment:
This is a bit odd.  Some systems pass in the objects passed out by scale_up and some systems expect worker addresses.  Adaptive expects worker addresses.

Reply:
Presumably, we already have the machinery to handle translating between address and "worker ids"? I think this would be useful in the job-queue context.